src/Symfony/Bundle/SecurityBundle/Resources/views/Collector/security.html.twig

```diff
@@ -159,10 +159,12 @@
                     <span class="value">{{ include('@WebProfiler/Icon/' ~ (collector.firewall.stateless ? 'yes' : 'no') ~ '.svg') }}</span>
                     <span class="label">Stateless</span>
                 </div>
+                {% if collector.authenticatorManagerEnabled == false %}
```

Comment:
I'd rather write it as `{% if not collector.authenticatorManagerEnabled %}`

Reply:
done https://github.com/symfony/symfony/pull/41139/commits/9dc446038fcbb27d479c217201d1cffe9a8853b8